### PR TITLE
Edit of Swedish translation

### PR DESCRIPTION
**Pull request by deob83**
I added translations of most (but not all) of the strings which weren't already translated into Swedish and fixed some translations which were marked as fuzzy or resulted in validation errors.

### CODE REVIEW

File: po/sv.po
Comment (der-lyse) [inline po/sv.po:1676]: That translation somehow got lost completely.

**Review by der-lyse — CHANGES_REQUESTED** [review]
Tack så mycket! I don't speak Swedish, so I could just do a basic plausibility check, I looked at punctuation, format string arguments and such. I probably found three small things, other than that this looks good to me. Once addressed, just merge. :-)

File: po/sv.po
Comment (Minoru) [inline po/sv.po:391]: There's too many newlines at the end of this message:

```suggestion
msgstr "FEL: Du måste ställa in `freshrss-url` för att använda FreshRSS\n"
```

File: po/sv.po
Comment (Minoru) [inline po/sv.po:1997]: Translation here got lost, too.

File: po/sv.po
Comment (Minoru) [inline po/sv.po:2027]:
```suggestion
msgstr "Kunde inte öppna låsfil `%s': %s"
```

File: po/sv.po
Comment (Minoru) [inline po/sv.po:2036]: Is this colon really needed? (It might be — I don't know Swedish.)

```suggestion
msgstr "Misslyckades att skriva PID till låsfil '%s': %s"
```

**Review by Minoru — COMMENTED** [review]
Thank you @deob83! Just like Lyse, I don't speak Swedish, but I think I found a couple more small formatting mistakes.

By the way, would you like to get added to the "Translators" team in Newsboat organization on GitHub? That way, you'll get pings two weeks before the release, so you have plenty of time to update the translation ;)

**Comment by Minoru** [comment]
Today is the release day, so I'll just apply the suggestions that make sense to me and merge this. You can fix the unresolved comments with a new PR.

**Comment by Minoru** [comment]
Oh, apparently you've forbidden the maintainers (i.e. me) to edit the branch from which you made the PR. I'll merge these changes manually then ;)

File: po/sv.po
Comment (deob83) [reply po/sv.po:1676]: The previous translation was completely wrong (it said "Error while reading feed"), but I wasn't happy with the translation I could come up with, so I just removed the previous translation. I should have mentioned that - I apologize. I'll fix it in a coming PR.

File: po/sv.po
Comment (deob83) [reply po/sv.po:1997]: In this case as well, the previous translation was completely wrong (it said "attribute not found"), so I removed it. Again, I should have mentioned that. I apologize! But I'll translate it in a coming PR.

File: po/sv.po
Comment (deob83) [reply po/sv.po:2036]: No, it's not needed - thanks!

**Comment by deob83** [comment]
> Oh, apparently you've forbidden the maintainers (i.e. me) to edit the branch from which you made the PR. I'll merge these changes manually then ;)

@Minoru: Sorry about that: I'm still not that experienced with GitHub. I'll make sure not to do that next time!

**Comment by deob83** [comment]
> Thank you @deob83! Just like Lyse, I don't speak Swedish, but I think I found a couple more small formatting mistakes.
> 
> By the way, would you like to get added to the "Translators" team in Newsboat organization on GitHub? That way, you'll get pings two weeks before the release, so you have plenty of time to update the translation ;)

@Minoru Yes, thanks, that would be great. :smiley:

**Comment by Minoru** [comment]
@deob83 Cool! I've just sent you an invite.

**Pull request merged**
